src/programs/match_template/match_template.cpp

```
@@ -42,9 +42,9 @@ WX_DEFINE_OBJARRAY(ArrayOfAggregatedTemplateResults);
 
 // nasty globals to track histogram size
 
```

I think there is something like a 4% performance hit to this, which is why I shrunk the range back in commit 4fc6328f554d

If I understand right, the problem is observed in platinum, so my question is whether or not we want to reflect these in the histogram or ignore them?

In the CPU code, out of range errors are ignored entirely:

https://github.com/bHimes/cisTEM_downstream_bah/blob/15a640a461dce6bb2118f89578e354b58a5e34dd/src/programs/match_template/match_template.cpp#L1129-L1132

In the GPU code, they are supposed to be stacked on either the lowest or highest bin

https://github.com/bHimes/cisTEM_downstream_bah/blob/15a640a461dce6bb2118f89578e354b58a5e34dd/src/gpu/Histogram.cu#L48-L50

1) So there are three options, ignore, truncate, or expand the range.
2) Then there is the question of why you are seeing a crash. I suspect either the MIN or MAX macro to be to blame. 

Once we answer (1) we can decide what do do, okay @jojoelfe ?

---

My initial guess would be to ignore. These locations will very probably not be biologically relevant and we don't want them to mess up any statistics we get from the histogram.